dgraph/cmd/alpha/run.go

```diff
@@ -544,23 +549,32 @@ func setupLambdaServer(closer *z.Closer) {
 			case <-closer.HasBeenClosed():
 				return
 			case <-ticker.C:
-				timestamp := time.Now().UnixNano()
-				for _, l := range lambdas {
+				healthCheck := func(l *lambda) {
```

Comment:
We should move this function definition outside the for...select

Reply:
Done